common/src/main/java/org/arpha/dto/media/response/FileResponse.java

```diff
@@ -0,0 +1,21 @@
+package org.arpha.dto.media.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.arpha.dto.media.enums.UploadType;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class FileResponse {
+
+    private long id;
+    private String fileName;
+    private long size;
+    private long targetId;
+    private String fileType;
+    private UploadType targetType;
+    private String link;
```

Comment:
Consider incapsulating fileType and link into FileAccessLink (I've just made up this class name) entity or something like that. Also consider adding the accessType field to this response so that the frontend is able to understand which kind of request it is supposed to send